Set initial_RED to 1

Drachenkaetzchen opened the pull request:
People complain about their Neopixels not working, not knowing that they need to call the macros before anything happens.

This PR is to set `initial_RED` to 1 to have a better out-of-the-box experience and less trouble for us supporters trying to figure out if the wiring is wrong or if people just don't know about the macros.

Drachenkaetzchen committed:
Set initial_RED to 1

People complain about their Neopixels not working, not knowing that they need to call the macros before anything happens.

This PR is to set `initial_RED` to 1 to have a better out-of-the-box experience and less trouble for us supporters trying to figure out if the wiring is wrong or if people just don't know about the macros.

wile-e1 commented:
Would having the `initial_color` set to Green=1?   Most of  the LEDs out there are GRB/GRBW.  If everything on startup is correct, and you get green.....  green means good?

Another option would be to set the `initial_color` to G=1 R=0 B=1.  If you get Cyan, everything is good and you have the right color order.  If you get Magenta, you have the the G and R backwards in the color order.  and if you don't get Magenta, something else is wrong.

Drachenkaetzchen commented:
That's an excellent idea!

Drachenkaetzchen committed:
Changed default to MAGENTA 

Also added a small comment section which tells the user which color format to use depending on the actual color they get.

wile-e1 commented on Klipper_Macros/stealthburner_leds.cfg:
```diff
@@ -5,7 +5,17 @@
+# for Neopixels with a dedicated white LED. On startup, all three SB LEDs should light up.
+#
+# If you get random colors across your LEDs, change the color_order to GRB and restart. Then
+# ignore the W for each suggested color_order in the next paragraph.
```
Maybe change "ignore" to "omit"

Drachenkaetzchen committed:
Fixed wording (ignore->omit)
